actors/market/src/lib.rs

```diff
@@ -304,10 +308,10 @@ impl Actor {
 
             // drop deals with insufficient lock up to cover costs
             let client_id = client.id().expect("resolved address should be an ID address");
-            let lockup = total_client_lockup.entry(client_id).or_default();
-            *lockup += deal.proposal.client_balance_requirement();
+            let mut client_lockup = total_client_lockup.entry(client_id).or_default().clone();
```

Comment:
If there is no lockup for the client, the current code will:

1. Insert 0.
2. Return a pointer to that zero.
3. Clone it.

Maybe that's fine... but it's weird. Better to do something like:

```suggestion
            let mut client_lockup = total_client_lockup.get(client_id).cloned().unwrap_or_default();
```

Reply:
`get(&client_id)`, I think, but yes! thank you!